### ISSUE TITLE
[Suggestion] Add a light-sensor-sensitive play/pause function (as user option)

### ISSUE THREAD

**Issue by Wikinaut** [segler-alex/RadioDroid#684]
In my household, we have several RadioDroids connected to our network, but only very few are actual playing.

In otder to save bandwidth and our carbon dioxid (environment) footprint we always have to stop transmission manually by pressing the pause/play button.

My idea is now to have a light-sensitive automatic function, 2 light level thresholds: one to start playing, one to pause playing, as a user-option of course.

**Comment by morckx** [comment]
@Wikinaut : Good idea also for my wall radios, thanks! However, you can do this easily with Tasker (Light level profile -> executing a Media Control Pause task). I think this is really rather a Tasker task.

BTW: I have rebased my [wall-radio-branch](https://github.com/morckx/RadioDroid/tree/wall-radio) on 0.81.

**Comment by Wikinaut** [comment]
Tasker?  AFAIK: not open source -> No way.

**Comment by werman** [comment]
There is an open source app with similar aims https://github.com/renyuneyun/Easer, however it doesn't support light level as a trigger at the moment.

> I think this is really rather a Tasker task.

I would agree that it is a task for other apps to do.

**Comment by morckx** [comment]
> Tasker? AFAIK: not open source; not free -> No way.

Good point :) Tasker, is so good and so well maintained since more than 10 years (?) that I had to make an exception for it. Maybe Easer is a possible alternative, but I haven't tried it.
